js/control/TrackAnalysis.js

```diff
@@ -279,29 +302,35 @@ BR.TrackAnalysis = L.Class.extend({
      * @returns {Object}
      */
     sortAnalysisData(analysis) {
-        var analysisSortable = {};
-        var result = {};
+        const analysisSortable = {};
+        const result = {};
 
-        for (var type in analysis) {
+        for (const type in analysis) {
             if (!analysis.hasOwnProperty(type)) {
                 continue;
             }
 
             result[type] = {};
             analysisSortable[type] = [];
 
-            for (var name in analysis[type]) {
+            for (const name in analysis[type]) {
                 if (!analysis[type].hasOwnProperty(name)) {
                     continue;
                 }
                 analysisSortable[type].push(analysis[type][name]);
             }
 
-            analysisSortable[type].sort(function (a, b) {
-                return b.distance - a.distance;
-            });
+            if (type === 'maxspeed') {
```

Comment:
nit: probably would have used a switch case for this but not a blocker at all :)